Makefile

```diff
@@ -454,7 +454,7 @@ $$(PYTHON_DIR-$(target))/Makefile: \
 $$(PYTHON_DIR-$(target))/python.exe: $$(PYTHON_DIR-$(target))/Makefile
 	@echo ">>> Build Python for $(target)"
 	cd $$(PYTHON_DIR-$(target)) && \
-		make all \
+		make CFLAGS='-I../openssl/$$(SDK-$(target))/include -I../xz/$$(SDK-$(target))/include -I$$(SDK_ROOT-$(target))/usr/include/ffi' all \
```

Comment:
ℹ️ Not sure if this is the best approach to fix the build errors complaining about missing include files for openssl, xz, and ffi…